### PR TITLE
Modif css global pour de vrai

### PR DESCRIPTION
**Pull request by Marie-Garde**

### CODE REVIEW

**Review by Gauthier-Lhomme — DISMISSED** [review]

**Comment by Gauthier-Lhomme** [comment]
Fixe some css error for desktop version : 
-Font style
-Styled css
-Add link icon and title

Styled css for mobile version : 
-customer area
-professional space

-Add reducer token client
-Add projector device
-Add  missing environment variable
-Add legal notice Content

File: src/index.js [hunk 1]
```diff
@@ -7,13 +7,15 @@ import { BrowserRouter } from "react-router-dom";
...
+import "./index.css";
+
 ReactDOM.render(
   <React.StrictMode>
     <BrowserRouter>
       <Provider store={store}>
         <App />
-      </Provider>
-    </BrowserRouter>
+      </Provider>{" "}
```
Comment (Gauthier-Lhomme) [inline src/index.js]: delete {" "}

File: src/index.js [hunk 2]
```diff
@@ -7,13 +7,15 @@ import { BrowserRouter } from "react-router-dom";
...
+    </BrowserRouter>{" "}
```
Comment (Gauthier-Lhomme) [inline src/index.js]: delete {" "}

**Review by Gauthier-Lhomme — DISMISSED** [review]

**Review by Gauthier-Lhomme — DISMISSED** [review]

**Review by Gauthier-Lhomme — DISMISSED** [review]

File: src/components/InscriptionSignUp.jsx [hunk 3]
```diff
@@ -38,8 +38,10 @@ export default function useForm(validateInfo) {
     delete values.password2;
     api.post("/auth/signup", values).then((res) => {
       setValues(res.data);
-      dispatch({ type: "SET_ID", setId: res.data.idClient });
-      dispatch({ type: "SET_INFOS", setInfos: res.data.infosClients });
+      dispatch({ type: "SET_ID", test: res.data.idClient });
+      dispatch({ type: "SET_INFOS", test: res.data.infosClients });
+      dispatch({ type: "SET_INFOS", test: res.data.tokenClient });
```
Comment (LoicBrassart) [inline src/components/InscriptionSignUp.jsx]: - "test" is a bad key name
- You dispatch the same action twice in a row, it doesn't seem like a good idea

**Review by LoicBrassart — DISMISSED** [review]
... Yeah, no, I won't read this, it's late 
I trust you read it among yourselves

**Review by Thoms-C — APPROVED** [review]
seems ok !

**Review by LoicBrassart — APPROVED** [review]
Still won't read

**Pull request merged**